src/js/components/AuthorSelector.js

```
@@ -3,232 +3,232 @@
+		{ label: __('Select an author…', 'author-profile-blocks'), value: '' },
+		...filteredAuthors.map(author => ({
+			label: author.name,
+			value: author.id.toString()
+		}))
+	];
 
-    // Handle selection change
-    const handleAuthorChange = (authorId) => {
-        setSelectedAuthorId(authorId);
+	// Handle selection change
+	const handleAuthorChange = (authorId) => {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)